guava/src/com/google/common/base/Platform.java

```diff
@@ -58,10 +58,22 @@ static boolean stringIsNullOrEmpty(@Nullable String string) {
     return string == null || string.isEmpty();
   }
 
+  /**
+   * Returns the string if it is not null; an empty string if it is null
```

Comment:
Nitpick: can you add a period at the end of this sentence, and the sentence on the next method?